gitclone/git.go

```diff
@@ -107,7 +111,18 @@ func getDiffFile(buildURL, apiToken string, prID int) (string, error) {
 
 func run(c *command.Model) error {
 	log.Infof(c.PrintableCommandArgs())
-	return c.SetStdout(os.Stdout).SetStderr(os.Stderr).Run()
+	var buffer bytes.Buffer
+	outWriter := io.MultiWriter(os.Stdout, &buffer)
+
+	err := c.SetStdout(outWriter).SetStderr(outWriter).Run()
+	if err != nil {
+		if errorutil.IsExitStatusError(err) {
+			return errors.New(buffer.String())
+		}
+		return fmt.Errorf("command execution failed, %s", err)
+	}
+
+	return nil
 }
 
 func output(c *command.Model) (string, error) {
```

Comment:
this function should be also updated as in case of an error we just print the error

Reply:
also `listBranches` function could also be updated